Drift/Private/DriftBase.cpp

```diff
@@ -2450,6 +2450,49 @@ void FDriftBase::ConfigureSettingsSection(const FString& config)
     }
 }
 
+void FDriftBase::InitDriftClientConfigs()
+{
+    auto request = GetGameRequestManager()->Get(driftEndpoints.client_configs, HttpStatusCodes::Ok);
+    request->OnResponse.BindLambda([this](ResponseContext& context, JsonDocument& doc)
+    {
+        FDriftClientConfigListResponse ConfigList;
+
+        if (!JsonArchive::LoadObject(doc, ConfigList))
+        {
+            FString Error;
+            context.error = TEXT("Failed to parse client config response");
+            DRIFT_LOG(Base, Error, TEXT("Failed to parse client configs from drift-base. Error: %s"), *Error);
```

Comment:
Isn't 'Error' always uninitialized here?

Reply:
Yes, that's a mistake on my part. Will fix

Reply:
Fixed in latest